doc/tor.md

```diff
@@ -216,10 +216,30 @@ for normal IPv4/IPv6 communication, use:
 
     ./bitcoind -onion=127.0.0.1:9050 -externalip=7zvj7a2imdgkdbg4f2dryd5rgtrn7upivr5eeij4cicjh65pooxeshid.onion -discover
 
-## 4. Privacy recommendations
+## 4. Security recommendations
 
-- Do not add anything but Bitcoin Core ports to the onion service created in section 3.
+Do not add anything but Bitcoin Core ports to the onion service created in section 3.
   If you run a web service too, create a new onion service for that.
   Otherwise it is trivial to link them, which may reduce privacy. Onion
   services created automatically (as in section 2) always have only one port
   open.
+  
+**Sybil Attacks**
+
+  If you only make random Tor connections, you're much more vulnerable to Sybil attacks.
+  As Tor addresses may be created at no cost, an attacker can potentially flood the network with many Tor
+  nodes and receive all of the outbound Tor connections an `onlynet=onion` node makes.
+
+  This is significantly less a concern if you make `-addnode` connections to trusted peers
+  (even if they're onion addresses). It's also alleviated with IPv4/IPv6 connections (especially when using the `-asmap`
+  configuration option) due to the cost of obtaining IPs in many networks.
+
+  If all of your connections are controlled by a Sybil attacker, they can easily prevent you from seeing confirmed
+  transactions and, with more difficulty, even trick your node into falsely reporting a transaction as confirmed
+  on the blockchain with the most cumulative “chainwork”.
```

Comment:
When describing what a Sybil attack is, would it be be better to link to an external source like https://en.bitcoin.it/wiki/Weaknesses#Sybil_attack which is more exhaustive, rather than trying to reproduce it here?

Reply:
I don't think we should be linking to external sources in our docs (and I think that's been our policy previously)

Reply:
Would invert the order of the above two paragraphs so that the addnode one is after the Sybil attack description, perhaps mention that a connection to a single honest peer is enough to thwart an attempted eclipse, and perhaps bring I2P and CJDNS into the discussion -- bitcoind users have more options than clearnet and Tor.

Reply:
> I don't think we should be linking to external sources in our docs (and I think that's been our policy previously)

We already link to https://en.bitcoin.it/wiki/Weaknesses#Sybil_attack in doc/i2p.md.

Reply:
I know links can break, but we can't really copy-paste the whole wisdom of the Internet into the `doc/` directory. Maybe summarize in 1 sentence what is a Sybil attack and mention e.g. "for more information see https://..." so that if the link breaks, it would not render our doc useless. No strong opinion.